### ISSUE TITLE
Crash on enter press for EditTexts with no tokens

### ISSUE THREAD

**Issue by markrietveld** [splitwise/TokenAutoComplete#116]
The `TokenCompleteTextView` in question is configured to allow free-form input using `performBestGuess(false)`. Doing a null check for `selectedObject` on that line avoids the crash. I don't know if it's the right thing to do overall

```
java.lang.NullPointerException
            at com.tokenautocomplete.TokenCompleteTextView.replaceText(TokenCompleteTextView.java:729)
            at com.tokenautocomplete.TokenCompleteTextView.performCompletion(TokenCompleteTextView.java:420)
            at com.tokenautocomplete.TokenCompleteTextView.handleDone(TokenCompleteTextView.java:440)
            at com.tokenautocomplete.TokenCompleteTextView.onEditorAction(TokenCompleteTextView.java:498)
            at android.widget.TextView.onEditorAction(TextView.java:4236)
            at com.android.internal.widget.EditableInputConnection.performEditorAction(EditableInputConnection.java:138)
            at android.view.inputmethod.InputConnectionWrapper.performEditorAction(InputConnectionWrapper.java:98)
            at com.android.internal.view.IInputConnectionWrapper.executeMessage(IInputConnectionWrapper.java:297)
            at com.android.internal.view.IInputConnectionWrapper$MyHandler.handleMessage(IInputConnectionWrapper.java:77)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:136)
            at android.app.ActivityThread.main(ActivityThread.java:5001)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:515)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
            at dalvik.system.NativeStart.main(Native Method)
```

**Comment by mgod** [comment]
hmmm, does your ```defaultObject``` return a non-null result?

**Comment by mgod** [comment]
From what I can see, if I do the null check, it doesn't clear the text and then no longer makes any completions possible. I think it might make more sense to add some documentation to defaultObject and enforce non-null results there?

**Comment by mgod** [comment]
This is fixed in [v2.0](https://github.com/splitwise/TokenAutoComplete/releases/tag/v2.0)
